components/analytics/team_analytics/team_analytics.tsx

```diff
@@ -75,46 +97,46 @@ export default class TeamAnalytics extends React.PureComponent {
         this.props.actions.getTeams(0, 1000);
     }
 
-    componentDidUpdate(prevProps, prevState) {
+    componentDidUpdate(prevProps: Props, prevState: State) {
         if (this.state.team && prevState.team !== this.state.team) {
             this.getData(this.state.team.id);
         }
     }
 
-    getData = async (id) => {
+    getData = async (id: string) => {
         AdminActions.getStandardAnalytics(id);
         AdminActions.getPostsPerDayAnalytics(id);
         AdminActions.getBotPostsPerDayAnalytics(id);
         AdminActions.getUsersPerDayAnalytics(id);
-        const {data: recentlyActiveUsers} = await this.props.actions.getProfilesInTeam(id, 0, General.PROFILE_CHUNK_SIZE, 'last_activity_at');
-        const {data: newUsers} = await this.props.actions.getProfilesInTeam(id, 0, General.PROFILE_CHUNK_SIZE, 'create_at');
+        const { data: recentlyActiveUsers } = await this.props.actions.getProfilesInTeam(id, 0, General.PROFILE_CHUNK_SIZE, 'last_activity_at');
+        const { data: newUsers } = await this.props.actions.getProfilesInTeam(id, 0, General.PROFILE_CHUNK_SIZE, 'create_at');
 
         this.setState({
             recentlyActiveUsers,
             newUsers,
         });
     }
 
-    handleTeamChange = (e) => {
+    handleTeamChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
         const teamId = e.target.value;
 
         let team;
-        this.props.teams.forEach((t) => {
+        this.props.teams.forEach((t: Team) => {
             if (t.id === teamId) {
                 team = t;
             }
         });
 
         this.setState({
-            team,
+            team: null, //! double check this
```

Comment:
In ES6, if the variable name matches the key name in an object declaration, we can simply pass the key name.
This works out to the same as:
```
this.setState({
            team: team,
});
```